src/Uno.Extensions.Templates/content/unoapp-extensions/Directory.Build.props

```diff
@@ -41,9 +45,16 @@
 		<SupportedOSPlatformVersion Condition="$(IsWinAppSdk)">10.0.18362.0</SupportedOSPlatformVersion>
 		<TargetPlatformMinVersion Condition="$(IsWinAppSdk)">10.0.18362.0</TargetPlatformMinVersion>
 	</PropertyGroup>
+	<!--#if (useNetSix)-->
 
 	<ItemGroup>
 		<!--This override is used to validate the use of specific version of the C# Compiler-->
+		<!--#if (cpm)-->
```

Comment:
Should this be a computed value eg useCPM instead of cpm?

Reply:
if it was computed it would simply equal cpm.. not sure what value we would get from it.